doc/api/esm.md

```diff
@@ -289,9 +292,10 @@ const { default: barData } =
 Node.js supports the following `type` values, for which the attribute is
 mandatory:
 
-| Attribute `type` | Needed for       |
+| Attribute `type` | Used with        |
 | ---------------- | ---------------- |
 | `'json'`         | [JSON modules][] |
+| `'jsonc'`        | [JSON modules][] |
```

Comment:
If we are to support this, the doc should indicate that `jsonc` is a non-standard import attribute value.